packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.18.1-alpha.2",
+  "version": "2.18.1-alpha.3",
```

Comment:
Since this is changing the default way core displays I am thinking we should make this a minor version bump